### PR TITLE
Preliminary doc re-structuring

### PR DESCRIPTION
**Pull request by brentfraser**
Added new headings
Re-organized existing doc headings

### CODE REVIEW

File: docs/howto/add-identify.md [hunk 1]
```diff
@@ -1,12 +1,114 @@
 # How-to add Identify
 
-Identify is supported on `wms`, `mapserver`, `wfs`, `mapserver-wfs`, and `ags-vector` map-source types.
+If you would like to have your map users to be able to get information on specific features using the Identify tool,
+you must:
+* ensure the `<map-source>` type is one of: `wms`, `mapserver`, `wfs`, `mapserver-wfs`, or `ags-vector`
```
Comment (klassenjs) [inline docs/howto/add-identify.md:5]: Shouldn't identify be supported on `geojson` too?

File: docs/howto/add-identify.md [hunk 2]
```diff
@@ -1,12 +1,114 @@
...
+                </div>
+            </div>
+            ]]>
+            </template>
+        </layer>
+```
+
+In the example above, there are a few things to note:
+* Customizing the look of the results is done using HTML syntax
+* GeoMoose substitutes values for references it knows about when encounters `{{ }}`
```
Comment (klassenjs) [inline docs/howto/add-identify.md:62]: Is there anything special that needs to be done to escapes/sanitize the data (or avoid escaping/sanitizing the data)?

File: docs/index.md [hunk 3]
```diff
@@ -2,17 +2,37 @@
 layout: default
 ---
 
-# Welcome to the GeoMoose Documentation
```
Comment (klassenjs) [inline docs/index.md:5]: This doesn't render well.  Looks kind of like if org-mode and markdown had a kid.

File: docs/howto/add-identify.md [hunk 4]
```diff
@@ -1,12 +1,114 @@
...
+            ]]>
+            </template>
+```
 
 ## The Firestations layer
 
-This how-to references the `firestations` source and `fire_stations` layer.
+This section references the `firestations` source and `fire_stations` layer.
 For more information on setting those up in your local demo, read the
 [How-to add a layer](./add-a-layer.md) guide.
```
Comment (klassenjs) [inline docs/howto/add-identify.md:113]: For consistency with the index: `the [Adding a layer](./add-a-layer.md) guide`

Did I mention that I hate markdown? I would have expected rubyists to have picked something that wasn't so actively hostile to DRY.

File: docs/howto/add-identify.md [hunk 5]
```diff
@@ -1,12 +1,114 @@
...
+                    <label>Name:</label>
+                    <a href=""
+                            onClick="window.open( 'http://www.census.gov/quickfacts/table/PST045215/{{properties.FIPS}},{{properties.COUNTYFIPS}}', 'Details','width=600,height=1000' ); return false"
+                            target="_blank">
+                        {{properties.COUNTYNAME}}</a>
+                </div>
+            </div>
+            ]]>
+            </template>
+```
```
Comment (klassenjs) [inline docs/howto/add-identify.md:107]:
```xml
<template src="..."/>
```
is not mentioned anywhere.

File: docs/howto/add-identify.md (hunk 1 above)
Comment (brentfraser) [reply docs/howto/add-identify.md:5]: Yes, but for some reason it wasn't listed in the doc.  That led me to today's debug session.  It should be listed in the doc, and the bug #522  needs to be fixed to make it work.

File: docs/howto/add-identify.md (hunk 2 above)
Comment (brentfraser) [reply docs/howto/add-identify.md:62]: Um, no?

File: docs/index.md (hunk 3 above)
Comment (brentfraser) [reply docs/index.md:5]: The "layout:default" was in the original doc.  I thought it was a artifact for the doc build system (which I eventually need to look into).  Should I take it out?

File: docs/howto/add-identify.md (hunk 4 above)
Comment (brentfraser) [reply docs/howto/add-identify.md:113]: Good point.  My intent with this PR was just to establish the top level layout and at least list some more sections so others could contribute.  All of the existing files need review and editing.  Huge task; just getting started...

File: docs/howto/add-identify.md (hunk 5 above)
Comment (brentfraser) [reply docs/howto/add-identify.md:107]: Yes. So much to doc, so little time...

File: docs/index.md (hunk 3 above)
Comment (klassenjs) [reply docs/index.md:5]: I mean the whole document renders strangely.  I don't know why or what layout:default does.  Click the "Display the rich diff" button for the file in the "Files Changed" tab of the PR.

It doesn't look like it has any clear structure.  

The ### nodes are all on one line.

File: docs/howto/add-identify.md (hunk 2 above)
Comment (brentfraser) [reply docs/howto/add-identify.md:62]: Very important points.  I can file a issue and hope the developer responds.

File: docs/index.md (hunk 3 above)
Comment (brentfraser) [reply docs/index.md:5]: Hopefully the latest commit to this PR resolves the link/header format issue.

File: docs/howto/add-identify.md (hunk 5 above)
Comment (theduckylittle) [reply docs/howto/add-identify.md:107]: src='..' is doc'd.

**Comment by theduckylittle** [comment]
@brentfraser GIthub does not show this has merged but it has been merged into master.

**Pull request closed without merging**